### PR TITLE
Rails Notes v2

### PR DESCRIPTION
**Pull request by lachlanjc**
- Adds Rails Notes link to homepage
- Runs Rails Notes workshop inside main `projects` C9 workspace
- Improves entire workshop

### CODE REVIEW

**Comment by zachlatta** [comment]
Boo to the yah. This is awesome.

Will remove the link you added – I already added one in #998.

Let's chat on Slack about how it went. Super curious to hear what students thought about it & how long it took them to get through it.

**Comment by lachlanjc** [comment]
Totally @zachlatta. Anything holding back the merge here?

File: workshops/rails_notes/README.md [hunk 1]
```diff
@@ -53,38 +68,44 @@ A **View** is the content of an actual webpage, like the HTML we've been writing
 <div><%= note.content %></div>
 ```
 
-A **Controller** connects *requests* (like for the `/notes` page) to your *view*, usually referencing a model. A Controller has (typically several) actions, which are different ways to interact with the `Note`. Here are the standard pages:
+A **Controller** connects *requests* (like for the `/notes` page to your *view*, usually referencing a model. A Controller has (typically several) actions, which are different ways to interact with the `Note`. Here are the standard pages:
```
Comment (cravend) [inline workshops/rails_notes/README.md]: There's a parenthesis missing from this line (after `/notes` page)

Comment (zachlatta) [inline workshops/rails_notes/README.md]: Fixed in 7cee21d.

**Comment by zachlatta** [comment]
🙌

**Pull request merged**